src/lib/scss/colors.scss

```diff
@@ -16,6 +16,7 @@ $cerulean: #00b6ed;
 $chateauGreen: #49af53;
 $cornflowerBlue: #6279e7;
 $cornflowerBlueTwo: #5871f4;
+$crimson:#E54158;
```

Comment:
no need to create add a new color. Use an existing color

Reply:
done.